include/net/xia_ether.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
+
+#include <linux/netdevice.h>
+#include <linux/etherdevice.h>
+#include <linux/err.h>
+#include <linux/netlink.h>
+#include <linux/rwlock.h>
+#include <linux/spinlock.h>
+
+/* Ethernet Principal. */
+#define XIDTYPE_ETHER (__cpu_to_be32(0x22))
+
+#ifdef __KERNEL__
+
+/* ETHER's virtal XID type. */
+int ether_vxt __read_mostly = -1;
```

Comment:
Variable `ether_vxt` is used in `net/xia/ppal_ether/main.c`, please move this declaration and its comment to `net/xia/ppal_ether/main.c`. Don't forget to limit the scope of `ether_vxt` with `static`.